std/assembly/array.ts

```diff
@@ -486,7 +486,7 @@ export class Array<T> extends ArrayBufferView {
     let size: i32 = 0;
     for (let i = 0; i < length; i++) {
       let child: T = load<T>(selfDataStart + (i << alignof<T>()));
-      size += child == null ? 1 : child.length;
+      size += child == null ? 1 : load<i32>(changetype<usize>(child), offsetof<T>("length_"));
```

Comment:
Hmm, I don't think this necessary. `child.length` should be optimize pretty well

Reply:
@dcodeIO mentioned we should do whatever is fastest. A load is faster than a function call.

Reply:
binaryen easily inline this call. See output. This only affect to `untouched` version. In the same time it looks not so clear as just length getter

Reply:
@willemneal mentioned this function call might be inlined. Just checked the optimized wat output and this is the case. This is faster for unoptimized builds, however.

Reply:
unoptimized builds shouldn't be optimized=) It mostly exists for easily read and verify results and of course for faster compilation)

Reply:
Alright, I'll revert this change.

Reply:
thanks)

Reply:
Is this all set?